llvm/test/CodeGen/AArch64/aarch64-reassociate-accumulators-sve.ll

```diff
@@ -1,4 +1,5 @@
-; RUN: opt -passes=loop-unroll %s -o - | llc -O3 - -mtriple=aarch64-unknown-unknown -mcpu=neoverse-v2 -o - | FileCheck %s
+; RUN: opt -passes=loop-unroll %s -o - | llc -O3 - -mtriple=aarch64-unknown-unknown \
+; RUN:   -mcpu=neoverse-v1 -mattr=+sve2 -o - | FileCheck %s
```

Comment:
I changed the RUN line here to match the spirit of the test, which is to reassociate accumulators. It was previously relying upon the scheduling of a neoverse-v2 CPU, but I can get the same effect by tuning for neoverse-v1 and enabling SVE2.

Reply:
If I leave the RUN line alone, then the test changes in such a way that it's no longer testing "reassociate chains of accumulation instructions into a tree" (https://github.com/llvm/llvm-project/pull/132728)